examples/gno.land/r/demo/profile/handlers.gno

```diff
@@ -0,0 +1,99 @@
+package profile
```

Comment:
I like this practice of defining all the rendering things in the same file.

Please, rename the file to `render.go` and move the `func Render` here.

Reply:
- https://github.com/gnolang/gno/pull/1983/commits/fadbc59ffcdaa3b38bec6606df64f86712b5edca